lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -33,9 +33,11 @@ sub verify_incident_repos ($url_handler, $incident_repos) {
 }
 
 sub detect_maintenance_update ($jobid, $url_handler, $settings) {
-    my $urls = collect_incident_repos($url_handler, $settings);
-    die "Current job $jobid will fail, because the repositories for the below updates are unavailable\n" . pp($urls)
-      if @$urls;
+    unless $settings->{PUBLIC_CLOUD_SKIP_MU}{
+        my $urls = collect_incident_repos($url_handler, $settings);
+        die "Current job $jobid will fail, because the repositories for the below updates are unavailable\n" . pp($urls)
+          if @$urls;
+    }
```

Comment:
Tidy won't like it. Why not just:
```suggestion
    return undef if $settings->{PUBLIC_CLOUD_SKIP_MU};
    my $urls = collect_incident_repos($url_handler, $settings);
    die "Current job $jobid will fail, because the repositories for the below updates are unavailable\n" . pp($urls)
      if @$urls;
    }
```

Reply:
Done, thanks :)